src/views/Recorder/RequestLog.tsx

```diff
@@ -70,14 +70,14 @@ export function RequestLog({
           )}
         </Flex>
       </Allotment.Pane>
-      <Allotment.Pane minSize={300} visible={selectedRequest !== null}>
-        {selectedRequest !== null && (
+      {selectedRequest && (
+        <Allotment.Pane minSize={300}>
           <Details
             selectedRequest={selectedRequest}
             onSelectRequest={setSelectedRequest}
           />
-        )}
-      </Allotment.Pane>
+        </Allotment.Pane>
+      )}
```

Comment:
I previously fixed a console warning by using the `visible` prop instead of conditionally mounting the pane, but that caused a flicker when switching to the requests tab (it's a [known issue](https://github.com/johnwalley/allotment/issues/818)).